tests/beta/queuing/conftest.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Pytest configuration file for PennyLane test suite.
+"""
+import os
+
+import pytest
+import numpy as np
+
+import pennylane as qml
+from pennylane.devices import DefaultGaussian
+
+import sys
+import importlib
+
+# defaults
+TOL = 1e-3
+TF_TOL = 2e-2
+
+
+class DummyDevice(DefaultGaussian):
+    """Dummy device to allow Kerr operations"""
+    _operation_map = DefaultGaussian._operation_map.copy()
+    _operation_map['Kerr'] = lambda *x, **y: np.identity(2)
+
+@pytest.fixture(scope="class")
+def import_beta():
+    """Import the beta version of queuing related modules."""
+    sys.modules["pennylane.operation"] = importlib.import_module('pennylane.beta.queuing.operation')
+    sys.modules["pennylane.measure"] = importlib.import_module('pennylane.beta.queuing.measure')
```

Comment:
A draft for an addition aiming to replace the loaded "original" modules with their "beta" counterparts